macros.in

```diff
@@ -1352,5 +1352,16 @@ end
     end
 }
 
+# buildsystem defaults
+%buildsystem_default_prep() %autosetup -p1
```

Comment:
I don't see this to be documented. Is it something one should use for something?

Reply:
Right, I'll need to document it. %buildsystem_default_* is just something a spec using Buildsystem: will fall back to if the buildsystem doesn't override it. So for the hands-free experience you'll want %autosetup as the default %prep, always. Whether this is useful for the other slots remains to be seen.

Reply:
Oh, another point regarding this: this will allow declaratively changing %prep behavior too, eg `BuildOption(prep): -n %{name}`. That wont work in this draft, yet, because the default prep macro isn't passing along arguments because I simply didn't think of this at all, but will update along with the docs for the next round.

Reply:
Documented in the last push.